ge/org.osate.ge.ba/src/org/osate/ge/ba/ui/properties/BehaviorTransitionPropertySection.java

```diff
@@ -145,7 +146,7 @@ public void refresh() {
 						.getEditingDomainFor(behaviorTransition);
 				final IProject project = ProjectUtil.getProjectForBoOrThrow(behaviorTransition);
 				final XtextResource xtextResource = getXtextResource(behaviorTransition)
-						.orElseThrow(() -> new RuntimeException("resource must be XtextResource"));
+						.orElseThrow(() -> new AadlGraphicalEditorException("resource must be XtextResource"));
```

Comment:
Could change resource to Resource to be consistent, but up to you

Reply:
I adjusted this since I needed to make the other change but in general things like this shouldn't hold up this branch because the inconsistent text wasn't something introduced in this branch.